otterdog/eclipse-bluechi.jsonnet

```diff
@@ -98,6 +98,32 @@ orgs.newOrg('eclipse-bluechi') {
         },
       ],
     },
+    orgs.newRepo('bluechi-ppa') {
+      allow_update_branch: false,
+      description: "PPA repository for BlueChi's .deb packages",
+      has_projects: false,
+      has_wiki: false,
+      homepage: "https://bluechi.readthedocs.io/en/latest/",
+      secret_scanning: "disabled",
+      secret_scanning_push_protection: "disabled",
+      squash_merge_commit_title: "PR_TITLE",
```

Comment:
seems redundant as its already the default setting afaict.

Reply:
Removed :heavy_check_mark: